keras_contrib/applications/densenet.py

```diff
@@ -297,7 +297,8 @@ def DenseNet(input_shape=None,
 def DenseNetFCN(input_shape, nb_dense_block=5, growth_rate=16, nb_layers_per_block=4,
                 reduction=0.0, dropout_rate=0.0, weight_decay=1E-4, init_conv_filters=48,
                 include_top=True, weights=None, input_tensor=None, classes=1, activation='softmax',
-                upsampling_conv=128, upsampling_type='deconv'):
+                upsampling_conv=128, upsampling_type='deconv', early_transition=False,
+                pooling='max', initial_kernel_size=(3, 3)):
```

Comment:
Shouldn't this be `transition_pooling` to match what is passed below in the __create_fcn_dense_net call?